ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -906,7 +998,7 @@ open class BottomCommandingController: UIViewController {
         struct BottomSheet {
             static let headerHeight: CGFloat = 66
             static let headerTopMargin: CGFloat = 8
-            static let headerLeadingTrailingMargin: CGFloat = 8
+            static let headerLeadingTrailingMargin: CGFloat = 16
```

Comment:
Should we use global tokens?

Reply:
I'll be working on tokenizing the control after this, so I figured just regular floats for now.